analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Utilities/SymbolReferenceAnalyzer/NamedType.vb

```diff
@@ -0,0 +1,10 @@
+﻿Imports System
+
+Public Class Sample
+
+    Public Sub Method()
+        Dim sample As Sample = New Sample()
+        Dim exception As Exception = New Exception()
```

Comment:
As the language is case insensitive, it's generally a bad practice to name the type and variable the same.
```suggestion
        Dim ex As Exception = New Exception()
```